client.go

```diff
@@ -0,0 +1,23 @@
+package appendlimit
+
+import (
+	"strconv"
+
+	"github.com/emersion/go-imap"
+)
+
+func MailboxStatusAppendLimit(status *imap.MailboxStatus) *uint32 {
+	val := status.Items[StatusAppendLimit]
+	if val == nil {
+		return nil
+	}
+
+	str := val.(string)
+	i64, err := strconv.ParseUint(str, 10, 32)
```

Comment:
There is a function for this: https://godoc.org/github.com/emersion/go-imap#ParseNumber

Reply:
Fixed in latest commit.